src/main/webapp/WEB-INF/view/register.jsp

```diff
@@ -50,7 +50,11 @@
       <label for="password">Password: </label>
       <br/>
       <input type="password" name="password" id="password">
-      <br/><br/>
+      <br/>
+      <label for="aboutMe">About Me (optional): </label>
+      <br/>
+      <textarea name="aboutMe" id="aboutMe" rows="5" cols="60"></textarea>
```

Comment:
I figured the user must register first before creating an aboutMe. Did we have to add this aboutMe label  and textbook in the register.jsp file? All of this is already stored in the profile.jsp. Would the code compile and run without adding this?

Reply:
textbox*

Reply:
The issue I was running into is that in RegisterServlet, it creates a new user with aboutMe as an argument to the constructor, but if the aboutMe is not on the registration page, it is always null, which throws an error. I figured since you made "aboutMe" an argument in the constructor, it might make sense to have that be set optionally when they register, and then they can alter it later on the Profile page.

Reply:
If you would rather not have the aboutMe box on the registration page then I can change it so aboutMe is set to an empty string, rather than set to null, and that won't cause an error.

Reply:
I see! It's alright the way it is now